README.md

```diff
@@ -59,6 +59,31 @@ Read the [User Guide](./docs/USER_GUIDE.md) for detailed documentation.
 
    and restart your shell.
 
+#### Fish
+1. Make sure that `git` is installed.
+2. Run this command in your terminal to download and install `krew`:
+
+    ```fish
+    begin
+      set -x; cd (mktemp -d) &&
```

Comment:
What if mktemp returns a path with space in it?
Shouldnt you quote this to prevent splitting?

Reply:
Hmm, good question. Actually, I am not that deep into fish, besides using it.
Taking a look.

Reply:
Just saw this while reading through fish.
https://github.com/fish-shell/fish-shell/issues/159#issuecomment-9533091.
It seems that subcommands in fish prevent space splitting.